### PR TITLE
add run_exports

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
sorry, i marked it as ready because i was getting impatient, but I think it is bes tto wait for the repodata patch.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,10 @@ source:
   sha256: 1c211b944ac83ef013fe91aee96c01289da4e3762c1e2f265449f3a964f8e4bc
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    # This is just a guess
+    - {{ pin_subpackage('svt-av1', max_pin='x.x') }}
```
Comment (carterbox) [inline recipe/meta.yaml:16]: I think @matthiasdiener should ask for clarification from upstream developers about [their ABI compatibility guarantees](https://github.com/AOMediaCodec/SVT-AV1/blob/6e87a1de98281840abebc030781780edd822bae5/CHANGELOG.md?plain=1#L14). If they are completely independent, then the pinning may need to be exact/all the way to minor version.
```suggestion
    - {{ pin_subpackage('svt-av1', max_pin='x.x.x') }}
```

Comment (hmaarrfk) [reply recipe/meta.yaml:16]: Do you want to pin hard and loosen later?

**Comment by hmaarrfk** [comment]
It seems that the repo patch went through, so once we agree on the pinning we can move forward

File: recipe/meta.yaml (hunk 1 above)
Comment (matthiasdiener) [reply recipe/meta.yaml]: Yeah, let's pin it more strictly for now I think.

**Comment by matthiasdiener** [comment]
Out of curiosity, why did you move this to Azure?

**Comment by hmaarrfk** [comment]
Travis was being slow and didn't start the job. We have more machines in count on azure

**Comment by carterbox** [comment]
@hmaarrfk has no time for slow bots! 😆

**Comment by matthiasdiener** [comment]
Thank you @hmaarrfk!

**Pull request merged**